### PR TITLE
SDKS-1676 & SDKS 1717

### PR DESCRIPTION
**Pull request by jeyanthanperiyasamy**
This Story fixes the delay broadcast issue  for few devices
And we stop making logout calls when the broadcast happen from another app 
Also added new integration test

### CODE REVIEW

File: forgerock-auth/src/test/java/org/forgerock/android/auth/SSOBroadcastReceiverIntegrationTests.kt [hunk 1]
```diff
@@ -0,0 +1,139 @@
...
+        assertNotNull(FRUser.getCurrentUser())
+
+        val testObject = SSOBroadcastReceiver()
+        val intent = Intent("org.forgerock.android.auth.broadcast.SSO_LOGOUT")
+        intent.putExtra("BROADCAST_PACKAGE_KEY", "com.test.forgerock")
+        testObject.onReceive(context, intent)
+
+        val sessionManagerObject: SessionManager = Config.getInstance().sessionManager
+
+        sessionManagerObject.hasSession()
```
Comment (witrisna) [inline forgerock-auth/src/test/java/org/forgerock/android/auth/SSOBroadcastReceiverIntegrationTests.kt:111]: What is the purpose of this calling hasSession()?

File: samples/authenticator/google-services.json [hunk 2]
```diff
@@ -0,0 +1,40 @@
+{
```
Comment (witrisna) [inline samples/authenticator/google-services.json:1]: Just to confirm, This file contains dummy data?

**Review by jeyanthanperiyasamy — COMMENTED** [review]
Addressed the changes @witrisna

**Review by witrisna — APPROVED** [review]

**Review by spetrov — APPROVED** [review]
LGTM 👍🏻

**Pull request merged**